cmd/oceantv/broadcast_states.go

```diff
@@ -637,50 +675,52 @@ func broadcastCfgToState(ctx *broadcastContext) state {
 	)
 	var newState state
 	switch {
```

Comment:
This switch is not very pretty haha

Reply:
Agreed. All ears if you have a better idea on how to do it hehe